src/GraphQL/Execution/QueryProcessor.php

```diff
@@ -25,11 +25,13 @@
 use GraphQL\Utils\TypeInfo;
 use GraphQL\Utils\Utils;
 use GraphQL\Validator\Rules\AbstractValidationRule;
-use GraphQL\Validator\ValidationContext;
 use GraphQL\Validator\Rules\QueryComplexity;
+use GraphQL\Validator\ValidationContext;
 use Symfony\Component\HttpFoundation\RequestStack;
 
-// TODO: Refactor this and clean it up.
+/**
+ * @todo Refactor this and clean it up.
+ */
```

Comment:
Unfortunately this does not a helpful class docblock comment make. So while it mutes the error it does not solve it.